src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -120,12 +122,14 @@ const ChangeRequestsTable: React.FC = () => {
         title={'Change Requests'}
         previousPages={[]}
         actionButton={
-          <ActionButton
-            link={routes.CHANGE_REQUESTS_NEW}
-            icon={faPlus}
-            text={'New Change Request'}
-            disabled={auth.user?.role === 'GUEST'}
-          />
+          <Link className={'row py-auto px-3 '} to={routes.CHANGE_REQUESTS_NEW} style={{ textDecoration: 'none' }}>
+            <Button variant="text">
```

Comment:
Make sure to also add **disabled** field as highlighted in ActionButton component in ActionButton.tsx file